src/components/member/tabs/contract-tab/InsuranceMandate.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { ButtonLink } from '../../../../../shared/hedvig-ui/button'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import { Contract } from '../../../../api/generated/graphql'
+import { ButtonSpacing } from './contract-item'
+
+export const InsuranceMandateComponent: React.FunctionComponent<{
+  contract: Contract
+}> = ({ contract }) => {
+  return (
+    <Card span={2}>
+      Insurance Mandate
```

Comment:
Depending on what it does, the CTA should probably be:
```suggestion
      Show Insurance Mandate
```
or 
```suggestion
      Download Insurance Mandate
```